appveyor.yml

```diff
@@ -0,0 +1,76 @@
+build: off
+
+branches:
+  only:
+    - master
+
+# ruby versions under test
+environment:
+  matrix:
+    - RUBY_VERSION: 23-x64
```

Comment:
I should cut this list down. Given that everything has passed on travis; we mostly only need appveyor to check that nothing Windows-specific is wrong.